internal/data/model/common_representation.go

```diff
@@ -13,11 +13,11 @@ import (
 // shared `Version` counter used for optimistic concurrency.
 type CommonRepresentation struct {
 	Representation
-	ResourceId                 uuid.UUID `gorm:"type:uuid;primaryKey"`
-	Version                    uint      `gorm:"type:bigint;primaryKey;check:version >= 0"`
-	ReportedByReporterType     string    `gorm:"size:128"`
-	ReportedByReporterInstance string    `gorm:"size:128"`
-	TransactionId              string    `gorm:"size:128;index:ux_common_reps_txid_nn,where:transaction_id IS NOT NULL AND transaction_id != '',unique"`
+	ResourceId                 uuid.UUID
```

Comment:
The whole purpose of a separate data model was to model the gorm tags, since we have the biz model that has the non gorm representation. I understand the reason for freezing a schema for versioning though. Can we just remove the separate types under model and just keep whatever is in steps (or store the versioned types in model), what is the impact?